pkg/ingester/ingester_v2.go

```diff
@@ -455,3 +488,93 @@ func (i *Ingester) closeAllTSDB() {
 	i.userStatesMtx.Unlock()
 	wg.Wait()
 }
+
+/* cleanup walks the TSDB directory for each user to determine if the TSDB should be closed.
+ * If
+ * - The directory is empty of blocks
+ * OR
+ * - The directory has every block marked as shipped
+ * AND
+ * - The TSDB hasn't been written to within the alloted time period
+ * THEN
+ * - Close the TSDB, stop the shipper routine, and delete the directory and it's contents.
+ */
+func (i *Ingester) cleanup() error {
+	ts := time.Now()
```

Comment:
Calling this variable `now` may help reading the following code.

Reply:
Changed.